libs/store/src/lib/effects/tournament.ts

```diff
@@ -0,0 +1,29 @@
+import { setReadyTournamentPayload } from '../payloads';
+import { Dispatch, RootState } from '../store';
+import {
+  AppTournament,
+  AppTournamentId,
+  AppTournamentState,
+} from '@razor/models';
+import { tournamentNotFound } from '../loggers';
+
+export const setReadyTournament = async (
+  dispatch: Dispatch,
+  payload: setReadyTournamentPayload,
+  state: RootState,
+): Promise<void> => {
+  const { tournamentId }: { tournamentId: AppTournamentId } = payload;
+
+  if (!state.game.tournamentsModel[tournamentId])
+    tournamentNotFound(dispatch, tournamentId, `While setting ready`);
```

Comment:
Why do we continue execution despite having failed an assertion?

Reply:
Fix in PR #14 under commit 6a024d6d4259d2be4fff29e45608b3a6b3735f75